README.md

```diff
@@ -1,2 +1,21 @@
-# documentation
-Official Yield Swap Documentation
+# Introduction
+
+## What is Yield Swap?
+
+Yield Swap is a decentralized exchange that allows users to swap interest rates.
+Users exchanging interest rates on Yield Swap are called [Payers](./payers.md)
+and [Receivers](./receivers.md). Payers pay the fixed interest rate of an asset
+in exchange for the floating interest rate provided by Receivers. Both user
+types do this in order to make a profit based on their respective investment
+strategies.
+
+## What problem does Yield Swap solve?
```

Comment:
General remark: it might make sense to use the term "yield" instead of "rate" in our context (although not 100% accurate, this seems to be more in line with DeFi lingo).

Reply:
We can go with "yield" if this fits better. Question on line 12 you commented on. Is there something here that needs fixing because the heading does not contain the word "rate".